tests/conftest.py

```diff
@@ -420,3 +421,15 @@ def teardown_hammer_defaults():
         assert 'organization_id' not in teardown.values()[0]['stdout']
 
     request.addfinalizer(teardown_hammer_defaults)
+
+
+@pytest.fixture(scope='function')
+def setup_old_foreman_tasks(ansible_module):
+    """ This fixture is for creating old foreman task."""
+    rake_command = "foreman-rake console <<< "
+    find_task = "'t = ForemanTasks::Task.where(state: \"stopped\").first;"
+    update_task = "t.started_at = t.started_at - 31.day;t.save(:validate => false)'"
+    try:
+        run(rake_command + find_task + update_task)
+    except Exception:
+        pass
```

Comment:
why just pass? good to raise error message.

Reply:
Because I don't want it to fail here even if any error occurs, I want it to continue executing the test.

Reply:
when it will fail? If it failed, test also will fail..right?

Reply:
Yes, test will fail but not here in setup.

Reply:
Add comment why we added it so that we will get to know afterwards